plugins/evaluation-plugin/intellij.evaluationPlugin.iml

```diff
@@ -32,5 +32,7 @@
     <orderEntry type="module" module-name="intellij.platform.core.ui" />
     <orderEntry type="module" module-name="intellij.platform.vcs" />
     <orderEntry type="module" module-name="intellij.platform.serviceContainer" />
+    <orderEntry type="module" module-name="intellij.java.psi" />
+    <orderEntry type="module" module-name="intellij.java.impl.refactorings" />
```

Comment:
unfortunately this won't work, language specific dependencies should be in language specific modules. let's discuss other options to use rename feature in the plugin in a meeting.